gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java

```diff
@@ -87,9 +87,24 @@ public boolean equals(Object obj) {
 
         private ProjectFileViewer viewer;
 
+        private final KeyCombination closeKeyCombination = new KeyCodeCombination(KeyCode.W, KeyCombination.CONTROL_DOWN);
```

Comment:
How do you chose this key combination? Is it something frequently used?

Reply:
it's the same KeyCombination used in Netbeans and Eclipse